ini/ncv1.ini

```diff
@@ -1,130 +1,127 @@
-046 = Mappy Arrangement - Name Entry
-047 = Mappy Arrangement - Name Entry Over
-
-101 = Xevious Arrangement - Area 1 (Restart)
-102 = Xevious Arrangement - Area 2 (Restart)
-103 = Xevious Arrangement - Area 3 (Restart)
-104 = Xevious Arrangement - Area 4 (Restart)
-10a = Xevious Arrangement - Area 5 (Restart)
-111 = Xevious Arrangement - Area 6 (Restart)
-116 = Xevious Arrangement - Area 7 (Restart)
-119 = Xevious Arrangement - Area 8 (Restart)
-11c = Xevious Arrangement - Area 9 (Restart)
-11f = Xevious Arrangement - Area 10 (Restart)
-122 = Xevious Arrangement - Area 11 (Restart)
-126 = Xevious Arrangement - Area 12 (Restart)
-12b = Xevious Arrangement - Area 13 (Restart)
-12d = Xevious Arrangement - Area 14 (Restart)
-12f = Xevious Arrangement - Area 15 (Restart)
-131 = Xevious Arrangement - Area 16 (Restart)
+[data]
+name     = Namco Classic Collection Vol.1
+driver   = Quattro
+type     = H8_ND
+filename = nc1sub.1c
+gain     = 1.0
+muterear = 1
+
+[wave.0]
+filename = nc1voice.7b
+length   = 0x200000
+position = 0x0
+
+[playlist]
+208 = Boot
+009 = Credit
+00d = Classics Opening
+330 = Galaga - Credit Sound
+331 = Galaga - Game Start Music
+339 = Galaga - Extend Sound
+33d = Galaga - Challenging Stage Start Music
+33f = Galaga - Challenging Start Clear Music
+33e = Galaga - Challenging Stage Perfect Music
+340 = Galaga - My Ship - Taken Away Sound
+342 = Galaga - Captive Error Sound
+341 = Galaga - My Ship - Regain Sound
+343 = Galaga - High Score Name Entry Music (1st Place)
+344 = Galaga - Name Entry Music (2nd-5th Place)
+206 = Galaga Arrangement - Insert Coin!
+20c = Galaga Arrangement - Game Start
+211 = Galaga Arrangement - Stage Indication
+203 = Galaga Arrangement - BGM - Stage 1 (Asteroid Zone)
+265 = Galaga Arrangement - BGM - Stage 2 (Nebula Zone)
+262 = Galaga Arrangement - BGM - Stage 3 (Galaxy Zone)
+24d = Galaga Arrangement - BGM - Stage 4 (Space-Plant Zone)
+268 = Galaga Arrangement - BGM - Stage 5 (Space-Flower Zone)
+26e = Galaga Arrangement - BGM - Stage 6 (Enemy-Comb Zone)
+22e = Galaga Arrangement - BGM - Challenging Stage Start
+250 = Galaga Arrangement - BGM - Challenging Stage
+230 = Galaga Arrangement - BGM - Challenging Stage Clear!
+235 = Galaga Arrangement - BGM - Challenging Stage Perfect!
+26b = Galaga Arrangement - BGM - Final Stage (Beat King-Galaspark!)
+272 = Galaga Arrangement - BGM - Indicating Your Score
+22a = Galaga Arrangement - BGM - Hi Score Name Entry (1st Place)
+22c = Galaga Arrangement - BGM - Name Entry (2nd-5th Place)
+249 = Galaga Arrangement - BGM - Continue
+24b = Galaga Arrangement - BGM - Game Over
+271 = Galaga Arrangement - BGM - Ending
+310 = Xevious - Credit Sound
+311 = Xevious - Start Music
+312 = Xevious - Game BGM
+31b = Xevious - Andor Genesis BGM
+loops = 10
+315 = Xevious - Extend Sound
+313 = Xevious - 1st Place Name Entry Music
+314 = Xevious - 2nd~5th Place Name Entry Music
+310 = Xevious Arrangement - Credit
+101 = Xevious Arrangement - BGM 1 (Area 1-4)

```

Comment:
The reason I put the "restart" versions at the bottom is because the area 2, 3, 4 etc music are already being played as part of a medley. So it matches the ingame and OST order more to just play it once  and then move on to the next track. I think I want to keep the titles from the arranged version as well, as they are still better than *no title*.

Reply:
I've made the changes you requested. Good ideas, actually.
Is there a way I can change this pull request or should I make a new one?
Again, I'm new to all this.

Reply:
You can just push more commits and they will show up.

Generally it's best to create a new branch before opening the PR as you can then keep your 'master' branch clean in case I (or someone else) decide to commit while it's still open.

Reply:
Okay, I'll keep that in mind.
Actually, I still can't figure out how to configure the ini for Assault.
Right now I have the ini set up like this:
```
[action.1]
t00 = 0x000c
t08 = 0x0007

[action.2]
t00 = 0x0031
t08 = 0x001c

0c = Unused 3 
time = 0 
action = 1 
loops = 24 
31 = Unused 4 
time = 0
action = 2
loops = 1
```
While this does play them simultaneously, using the "S" key only stops the C140 channels.
The YM2151 keeps playing. Unused 3 also plays two kick drum samples at the beginning for some reason, when there should only be one.
It would be helpful if there was a guide on making ini files and all of the different functions that Quattroplay supports.

Reply:
Try this to fix the duplicated sample:
<pre>
31 = Unused 3 
time = 0 
action = 1 
action = 0
loops = 24 
31 = Unused 4 
time = 0
action = 2
action = 0
loops = 1
</pre>

The S key does not stop the tracks because the song request action do not properly register the subsongs. It can be fixed in the code but a workaround is to simply play another track from the playlist and stop it normally.

Reply:
That worked, but now there's a very loud click in the left channel at the beginning of Unused 3.
I think it's coming from the kick drum sample which I never realized was in stereo.